### ISSUE TITLE
sublime Project and coverage to git ignore list

### ISSUE THREAD

**Issue by karlcow** [webcompat/webcompat.com#2364]
Coverage in python adds a folder:

```
	.coverage
```

Sublime Text allows local configuration in a project, but creates a folder too
```
	.sublime-project
```

Proposal to add them to the `.gitignore` or do we consider that we should add them to the project at a point. Both have pros and cons.

**Comment by Regaddi** [comment]
I would possibly never (say never never?) add the `.coverage` folder to the tracked files in git. They should be only processed during the build, but there's no need to have generated HTML/CSS in your repository (unless you want to provide `dist` files for easy inclusion).

Regarding the `.sublime-project`: it's a configuration file. I think it's fine to share it. I personally use VSCode (which also allows project configurations). I guess we shouldn't add too many config files/folders, because they're all in the project root...

**Comment by zoepage** [comment]
> I guess we shouldn't add too many config files/folders, because they're all in the project root...

I agree here. The `.eslint-config` should be enough for the editor someone is using.

> Proposal to add them to the .gitignore.

:+1:

> I would possibly never (say never never?) add the .coverage folder to the tracked files in git.

Also agree, but not experienced developers might. This is why I also second to add `.coverage` as well as `/coverage` to the .gitignore.
